doc/man1/openssl-rand.pod.in

```diff
@@ -14,12 +14,14 @@ B<openssl rand>
 [B<-hex>]
 {- $OpenSSL::safe::opt_engine_synopsis -}{- $OpenSSL::safe::opt_r_synopsis -}
 {- $OpenSSL::safe::opt_provider_synopsis -}
-I<num>
+I<num>[k|m|g|t]
```

Comment:
It seems to me we're beating around the bush here and what many people will actually want is just an option for infinite output, ala `dd if=/dev/urandom` (even if not truly infinite). Perhaps let people also just put `inf`/`infinite` here, or omit the argument entirely for a default random mode.

Reply:
+1 to support semi-infinite output, limited by SP 800-90C constraints. Better keyword would be `max` to make it clear that more Bytes cannot be generated. 

FYI - SP 800-90C limits DRBG output to 2⁶⁴ **bits** per instantiation. 

When preparing hard-drive for encryption, you want to fill it with random data first. In this usecase, it's very convenient to ask for more Bytes than needed, knowing that process is automatically stopped when the whole disk is full. Having keyword `max` would be user friendly.

Reply:
_(I should get in the habit of refreshing my page before responding; didn't see @jirka-h's reply)_

> FYI - SP 800-90C limits DRBG output to 2⁶⁴ bits per instantiation.

Which luckily coincides with SIZE_MAX on most platforms? So a cheap solution would be to just set `num=SIZE_MAX` in that case and not bother about querying RNG properties. Wouldn't that be good enough in practice?

Reply:
added check for special string 'max' in num value, setting scaled_num to SIZE_MAX to generate maximally allowed rng bytes.  Added note to pod file indicating max value and reasoning

Reply:
fixed capitalization, added 'max' string to produce maximal number of random bytes (2^64), documented limitation and reasoning in pod